### PR TITLE
🌐 Fix typos in German translation for `docs/de/docs/features.md`

### PR DESCRIPTION
**Pull request by 0xflotus**
I fixed some small errors in the german translation of *features.md*

### CODE REVIEW

File: docs/de/docs/features.md [hunk 1]
```diff
@@ -119,9 +119,9 @@ Die gesamte Validierung übernimmt das etablierte und robuste **Pydantic**.
 
 ### Sicherheit und Authentifizierung
 
-Sicherheit und Authentifizierung integriert. Ohne einen Kompromiss aufgrund einer Datenbank oder den Datenentitäten.
+Sicherheit und Authentifizierung sind integriert. Ohne einen Kompromiss aufgrund einer Datenbank oder den Datenentitäten.
```
Comment (rogerbrinkmann) [inline docs/de/docs/features.md:122]: Or:
`Integrierte Sicherheit und Authentifizierung. Ohne Kompromisse bei Datenbanken oder Datenmodellen.`

File: docs/de/docs/features.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ FastAPI enthält ein extrem einfaches, aber extrem mächtiges <abbr title='oft v
 * **Automatische Umsetzung** durch FastAPI.
 * Alle abhängigen Komponenten könnten Daten von Anfragen, **Erweiterungen der Pfadoperations-**Einschränkungen und der automatisierten Dokumentation benötigen.
 * **Automatische Validierung** selbst für *Pfadoperationen*-Parameter, die in den Abhängigkeiten definiert wurden.
-* Unterstütz komplexe Benutzerauthentifizierungssysteme, mit **Datenbankverbindungen**, usw.
+* Unterstützt komplexe Benutzerauthentifizierungssysteme, mit **Datenbankverbindungen**, usw.
```
Comment (rogerbrinkmann) [inline docs/de/docs/features.md:145]: The englisch version doesn't include a `with`:
`Unterstützt komplexe Benutzerauthentifizierungssysteme, **Datenbankverbindungen**, usw.`

**Review by rogerbrinkmann — APPROVED** [review]
Good changes, they are correct and necessary.

File: docs/de/docs/features.md (hunk 2 above)
Comment (0xflotus) [reply docs/de/docs/features.md]: adopted

File: docs/de/docs/features.md (hunk 1 above)
Comment (0xflotus) [reply docs/de/docs/features.md]: adopted

**Comment by rogerbrinkmann** [comment]
@0xflotus, thank's for you work. Let's keep it rolling. Just recently I added two new translations. #4517 and #4520. Maybe, if you can spare the time, you could return the favour and review them.

**Review by iron3oxide — APPROVED** [review]
Can't spot any change I wouldn't agree with.

**Comment by tiangolo** [comment]
Great, thank you @0xflotus! :rocket: 

And thanks for the reviews @rogerbrinkmann and @iron3oxide :cake:

**Pull request merged**